### DISCUSSION TITLE
HERO11 Labs 2.10.70

### DISCUSSION THREAD

**Discussion by dnewman-gpsw** [gopro/labs#265] [Announcements]
Major update for Labs.

The Release notes are updated here: https://gopro.github.io/labs/control/notes/

New firmware : https://gopro.github.io/labs

**Comment by andrerav** [comment]
Fantastic work! I updated today and tried to enable GCMP=1, however on my display it seems that the shutter and ISO is still affected by camera movement (I have EXPS enabled). Are there some special conditions needed for GCMP to work, or are the EXPS values captured before the GCMP hack takes effect?

Edit: I did a !RESET, and enabled only GCMP and EXPS, but still seeing the same result. I also did a few recordings with max ISO set to 6400, and the resulting clips seem to visually confirm that GCMP=1 does not have any effect. I also double checked that my camera is running the correct firmware version.

**Reply by dnewman-gpsw** [reply]
Seems a sloppy merge chopped that one feature, just the parsing for GCMP, as rest is there.  I removed it from the release notes, but it will return.

**Reply by andrerav** [reply]
Oh, so sorry to hear that, but I know how easily that can happen :) Do you think it will make a return before summer? 🙏
By the way, WIDE=1 is absolutely fire. I did a few shots today in quite challenging conditions, and grading the footage is just pure bliss. Even my neon pink handlebar grips are quite manageable now 🥰

**Reply by dnewman-gpsw** [reply]
Thank you, for those that understand WIDE, it is awesome, and much easier than Native white balance.

**Reply by HiStephenT** [reply]
Andrea’s did you use just WIDE=1 or combine it with say logb?

**Reply by andrerav** [reply]
@HiStephenT I combine it with LOGB=400. I've uploaded some sample footage here with all settings listed: https://www.youtube.com/@snokefilmlabs
